docs/ceph-csi-upgrade.md

```diff
@@ -98,12 +98,12 @@ For example, upgrading from 3.10 to 3.12 is not recommended.
 [release notes](https://github.com/ceph/ceph-csi/releases/tag/v3.12.0) before
```

Comment:
update the tag above as well? It pointing to 3.12.0

Reply:
> update the tag above as well? It pointing to 3.12.0

That refers to breaking changes in release notes. 
we usually have breaking changes only in major version 0th release.
hence, left it as it is.
Let me know if you still want to update it.

We refer to 3.10.0 release notes for breaking changes here too

https://github.com/ceph/ceph-csi/blob/release-v3.10/docs/ceph-csi-upgrade.md#upgrading-from-v39-to-v310